# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/components.conf
# Commit: 63b555e1f503
# Full Hash: 63b555e1f503db91a54604fd8cef88dfab7c0c7a
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2023-06-08 09:27:30
# Regressor Bug: 1834862
# File Overlap Count: 1
# Description:
#   Bug 1834862 - implement nsICryptoHash in rust r=jschanck,supply-chain-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D179727
# ==============================================================================

diff -r 6337ce701771 -r 63b555e1f503 security/manager/ssl/components.conf
--- a/security/manager/ssl/components.conf	Wed Jun 07 22:44:15 2023 +0000
+++ b/security/manager/ssl/components.conf	Wed Jun 07 23:03:21 2023 +0000
@@ -69,8 +69,8 @@
     {
         'cid': '{36a1d3b3-d886-4317-96ff-87b0005cfef7}',
         'contract_ids': ['@mozilla.org/security/hash;1'],
-        'type': 'nsCryptoHash',
-        'legacy_constructor': 'mozilla::psm::NSSConstructor<nsCryptoHash>',
+        'headers': ['/security/manager/ssl/crypto_hash/crypto_hash.h'],
+        'legacy_constructor': 'crypto_hash_constructor',
     },
     {
         'cid': '{45a5fe2f-c350-4b86-962d-02d5aaaa955a}',